indico/web/forms/widgets.py

```diff
@@ -294,6 +294,7 @@ def __init__(self, post_url=None, max_file_size=None, max_files=10, add_remove_l
             'parallelUploads': max_files,
             'handleFlashes': handle_flashes
         }
+        self.style = style
```

Comment:
Do we really want a freeform `style` argument instead of something like `lightweight=True`?